Refresh.GameServer/Endpoints/Game/CommentEndpoints.cs

```diff
@@ -30,6 +30,12 @@ public Response PostProfileComment(RequestContext context, GameDatabaseContext d
         GameUser? profile = database.GetUserByUsername(username);
         if (profile == null) return NotFound;
         
+        
+        int count = database.GetNotificationCountByUser(profile);
+        if (!profile.Equals(user) && count < 10) {
```

Comment:
I like the idea behind not spamming the user with notifications, but this seems a little arbitrary...

I think the end-goal here is for users to be able to toggle which notifications they want to receive. Maybe a set of 'channels', so like:
```csharp
enum NotificationChannel : byte
{
    UserHeart,
    UserMutual,
    ScoreOvertake,
    Comment,
    // etc...
}
```

And then you'd be able to fine-tune which of those you want to receive on the website.

I'm okay with just always sending a notification for now but this is something that should be addressed in a follow-up PR either by one of us, or you if you'd like.

Reply:
Sorry, you're right, I shouldn't have used a magic number like that. I like the idea of toggling notifications, I'll try to work on it soon.